pkg/services/ngalert/README.md

```diff
@@ -0,0 +1,87 @@
+# Next generation alerting (ngalert) in Grafana 8
+
+Ngalert (Next generation alert) is the next generation of alerting in Grafana 8.
+
+## Overview
+
+The ngalert package can be found in `pkg/services/ngalert` and has the following sub-packages:
+
+    - api
+    - eval
+    - logging
+    - metrics
+    - models
+    - notifier
+    - schedule
+    - sender
+    - state
+        - The state package does XYZ
+    - store
+    - tests
+
+## Scheduling and evaluation of alert rules
+
+The scheduling of alert rules happens in the `schedule` package. This package is responsible for managing the evaluation
+of alert rules including checking for new alert rules and stopping the evaluation of deleted alert rules.
+
+The scheduler runs at a fixed interval, called its heartbeat, in which it does a number of tasks:
+
+1. Fetch the alert rules for all (excluding disabled) organizations
```

Comment:
I would write something like "Fetch the alert rules for all organizations (excluding disabled ones)."
Aso, periods after sentences.